src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTarget.java

```diff
@@ -142,6 +142,7 @@
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
+import java.util.Stack;
 
 public class TextEditingTarget implements 
```

Comment:
Since TextEditingTarget is such a large class to begin with we have a suite of TextEditingTarget*Helper classes where we park orthogonal code. Could we move most of this code into a TextEdtingTargetReformatHelper?